violence/models.py

```diff
@@ -46,7 +46,4 @@ def tags_and_colors(self):
         )
 
     def is_valid(self):
-        if not self.when:
-            return False
-
-        return True
+        return self.when
```

Comment:
I kept the structure so we could add more `if`s later on. But as I forgot to document this choice, the suggestion is fair enough : )

What about `bool(self.when)` to be more semantic with the method name?